composer.json

```diff
@@ -1,16 +1,11 @@
 {
     "name": "b13/online-media-updater",
+	"version": "2.0.0",
```

Comment:
any reason why you add the "version"?

Reply:
I needed this for my local Dev environment. Will be switched again. Remember: This is a draft, I'm still developing ;-)